integration-tests/testconfig/vrfv2plus/overrides/new_env/soneium_sepolia_new_env_test_config.toml

```diff
@@ -0,0 +1,8 @@
+[Network]
+selected_networks = ["SONEIUM_SEPOLIA"]
```

Comment:
Is it actually working? Since its a new network and I dont see it specified in CTF [fw](https://github.com/smartcontractkit/chainlink-testing-framework/blob/e356557885b9528dcb201adc8a6b7d23a9ec4b8f/lib/networks/known_networks.go#L1010C1-L1078C41)

Or other way was to specify it in [default.toml](https://github.com/smartcontractkit/chainlink/blob/65924811dc53a211613927c814d7f04fd85439a4/integration-tests/testconfig/default.toml#L469-L481) config

Reply:
Also, dont forget to add default chain setting like [here](https://github.com/smartcontractkit/chainlink/blob/65924811dc53a211613927c814d7f04fd85439a4/integration-tests/testconfig/default.toml#L454-L466)

Reply:
It's actually not working. I had to specify the entire `[Network.EVMNetworks.SONEIUM_SEPOLIA]` in my local config. There's urls, and keys in the config so I'm not committing the changes. For this PR to work, the CTF networks has to be updated like you said.

Reply:
@leeyikjiun you need to put URLS into `~/.testsecrets` as per https://github.com/smartcontractkit/chainlink-testing-framework/blob/a1811529658da84a861b4bd966adafe23789f531/lib/config/README.md#test-secrets 

Also, recommend a [video](https://drive.google.com/file/d/15WwxwR3CkFlEX0K-RwlKYRQJan5yc4bh/view) on this if you have not seen it yet